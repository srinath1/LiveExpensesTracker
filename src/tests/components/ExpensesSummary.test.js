import React from 'react'
import { shallow } from 'enzyme'
import{ ExpensesSummary} from '../../components/ExpensesSummary'
test('should correctly render expenses with 1 expense',()=>{
    const wrapper=shallow(<ExpensesSummary expenseCount={1} expenseTotal={123}/>)
    expect(wrapper).toMatchSnapshot()

})
test('should correctly render expenses with multiple expense',()=>{
    const wrapper=shallow(<ExpensesSummary expenseCount={10} expenseTotal={123456}/>)
    expect(wrapper).toMatchSnapshot()
    
})