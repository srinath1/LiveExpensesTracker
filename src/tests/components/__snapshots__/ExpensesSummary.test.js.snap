// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render expenses with 1 expense 1`] = `
<div>
  <h1>
    Viewing 
    1
     expenses 
    $1.23
  </h1>
</div>
`;

exports[`should correctly render expenses with multiple expense 1`] = `
<div>
  <h1>
    Viewing 
    10
     expenses 
    $1,234.56
  </h1>
</div>
`;
